include/QuantumNumbers.h

```diff
@@ -82,6 +82,15 @@ class QuantumNumbers
 
     /// @}
 
+    /// \name Setters
+    /// @{
+
+    /// set Isospin * 2
```

Comment:
correct the doxy.

I think we should also add setters for everything, if we have for some.